src/test/java/org/trebol/operation/controllers/ControllersCorsTest.java

```diff
@@ -0,0 +1,235 @@
+		POST /data/sales/rejection;\
+		POST /data/sales/completion;\
+		GET,PUT /account/profile;\
+		GET /access;\
+		GET /access/*;\
+		GET /data/people;\
+		GET /data/billing_types;\
+		GET /data/sell_statuses;\
+		GET,POST,PUT,DELETE /data/customers;\
+		GET,POST,PUT,DELETE /data/images;\
+		GET,POST,PUT,DELETE /data/products;\
+		GET,POST,PUT,DELETE /data/product_lists;\
+		GET,POST,PUT,DELETE /data/product_list_contents;\
+		GET,POST,PUT,DELETE /data/product_categories;\
+		GET,POST,PUT,DELETE /data/sales;\
+		GET,POST,PUT,DELETE /data/salespeople;\
+		GET,POST,PUT,DELETE /data/shippers;\
+		GET,POST,PUT,DELETE /data/users;\
+		GET,POST,PUT,DELETE /data/user_roles;\
+		*/
+				
+		apiPaths = List.of(new ApiPath("/", "GET"),
+						new ApiPath("/public/about", "GET"),
+						new ApiPath("/public/receipt/TOKEN", "GET"),
+						new ApiPath("/public/checkout", "POST"),
+						new ApiPath("/public/checkout/validate", "POST", "GET"), // GET was missing
+						new ApiPath("/public/guest", "POST"),
+						new ApiPath("/public/login", "POST"),
+						new ApiPath("/public/register", "POST"),
+						new ApiPath("/data/sales/confirmation", "POST"),
+						new ApiPath("/data/sales/rejection", "POST"),
+						new ApiPath("/data/sales/completion", "POST"),		
+						new ApiPath("/account/profile", "GET", "PUT"),
+						new ApiPath("/access", "GET"),
+						new ApiPath("/access/APIROUTE", "GET"),
+						new ApiPath("/data/people", "GET"),
+						new ApiPath("/data/billing_types", "GET"),
+						new ApiPath("/data/sell_statuses", "GET"),
+						new ApiPath("/data/customers", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/images", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/products", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/product_lists", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/product_list_contents", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/product_categories", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/sales", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/salespeople", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/shippers", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/users", "GET", "POST", "PUT", "DELETE"),
+						new ApiPath("/data/user_roles", "GET", "POST", "PUT", "DELETE"));
+		
+		// Some ApiPaths with Invalid methods to test (if this fail, others should fail too)
+		apiPathsWithInvalidMethods = List.of(new ApiPath("/public/checkout", "GET"),
+											new ApiPath("/public/checkout/validate", "PUT"),
+											new ApiPath("/public/guest", "GET"),
+											new ApiPath("/public/login", "GET"),
+											new ApiPath("/public/register", "GET"),
+											new ApiPath("/data/sales/confirmation", "GET"),
+											new ApiPath("/data/sales/rejection", "GET"),
+											new ApiPath("/data/sales/completion", "GET"));
+	}
+	
+	// Arguments Providers
+	
+	// All paths with all methods and all allowed origins	
```

Comment:
Please turn this into a method JavaDoc comment
```suggestion
	/** 
	 * Provides all paths, all methods and all allowed origins as test method parameters.
	 */
```